requirement-generator/checkers/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Checkers } from './types';
```

Comment:
Can we rename this file to `requirementsList.ts` or whatever is more suitable? Reason being is that we want to make the adding more requirements data process as clear and seamless as possible